addon/components/listing.js

```diff
@@ -16,7 +16,7 @@ import ListingTable from '@lblod/ember-submission-form-fields/components/listing
 const ORDER = SHACL('order');
 
 export default class ListingComponent extends Component {
-  @tracked subForms = A();
+  @tracked subForms = [];
```

Comment:
I refactored this to a normal array already which solves the "'You attempted to update `subForms` on `ListingComponent`" issue (but we still run into others).